projects/sandbox/src/app/_models/progress-info.ts

```diff
@@ -64,13 +64,19 @@ export interface TierInfo {
   total: number;
 }
 
+export interface DatasetLog {
+  type: string;
+  message: string;
+}
+
 export interface Dataset {
   status: DatasetStatus;
```

Comment:
I see that `DatasetStatus` does not include the new value `FAILED`.
You should take it into account this new values. 
Perhaps put an error symbol next to the dataset name when it fails

Reply:
Done